docs/fundamentals-and-concepts/list-of-operations.mdx

```diff
@@ -685,20 +685,56 @@ Note that Soroban transactions can only contain one operation per transaction.
 
 Learn more in the [Soroban docs](https://soroban.stellar.org/docs/fundamentals-and-concepts/invoking-contracts-with-transactions#invokehostfunctionop).
 
-## Bump Footprint Expiration
+**Threshold**: Medium  
+**Result**: `InvokeHostFunctionResult`  
+**Parameters**:
+
+| Parameters | Type | Description |
+| --- | --- | --- |
+| Host Function | HostFunction | The host function to invoke |
+| Auth | Soroban Authorization Entry | Per-address authorizations for this host function. |
+
+**Possible errors**:
+
+| Error | Code | Description |
+| --- | --- | --- |
+| INVOKE_HOST_FUNCTION_MALFORMED | -1 | One or more of the inputs to the operation was malformed. |
+| INVOKE_HOST_FUNCTION_TRAPPED | -2 | The function invocation trapped in the Soroban runtime. |
+| INVOKE_HOST_FUNCTION_RESOURCE_LIMIT_EXCEEDED | -3 | The function invocation could not complete within the currently configured resource constraints of the network. |
+| INVOKE_HOST_FUNCTION_ENTRY_ARCHIVED | -4 | A ledger entry required for this function's footprint is in an archived state, and must be restored. |
+| INVOKE_HOST_FUNCTION_INSUFFICIENT_REFUNDABLE_FEE | -5 | The refundable Soroban fee provided was not sufficient to pay for the compute resources required by this function invocation. |
+
+## Extend Footprint TTL
 
 :::info
 
 Soroban is currently live on Testnet, so this operation is only usable on Testnet.
 
 :::
 
-Bump the expiration ledger of entries for Soroban smart contracts with the `BumpFootprintExpirationOp`.
+Extend the TTL of entries for Soroban smart contracts with the `ExtendFootprintTTLOp`. This operation Extends the TTL of the entries specified in the `readOnly` footprint so they will live at least `extendTo` ledgers from last closed ledger.
```

Comment:
Suggestion: it'd be nice to expand the acronym at least once! Also, it's a little confusing but even though `TTL` is relative, `extendTo` is an absolute ledger number.

```suggestion
Extend the time to live of entries for Soroban smart contracts with the `ExtendFootprintTTLOp`. This operation extends the TTL of the entries specified in the `readOnly` footprint of the transaction so that they will live at least until the `extendTo` ledger sequence number is reached.
```

Reply:
Thanks! Really good call here!